frontend/src/components/GoalsTable/GoalRow.js

```diff
@@ -231,31 +237,78 @@ function GoalRow({
                 svgLineTo={300}
               />
             )
-            : displayGoalTopics
-        }
-      </td>
-      <td>
-        <strong>{objectiveCount}</strong>
-        {' '}
-        Objective(s)
-      </td>
-      <td>
-        {
-          showContextMenu
+            : displayGoalTopics}
+        </td>
+        <td>
+          <a
```

Comment:
I know you didn't originate this pattern, but I think today might be a nice day to stop using <a> as a <button>